### DISCUSSION TITLE
Filter pull requests by subscription

### DISCUSSION THREAD

**Discussion by huehnerlady** [community/community#136946] [Pull Requests] [answered]
### Select Topic Area

Question

### Body

Hi,

I currently have the following filter to get an overview of the pull requests I need to review:

is:open is:pr review-requested:huehnerlady archived:false draft:false 

Sometimes I know another person will look at a pull request and for that I turn off notifications. 
It is requesting a review from me through a team though.
What I would like is to also get it out of my overview.

Any idea how I can achieve that?

**Comment by akhilsamvarghese** [comment]
Hi **huehnerlady**,

You can refine your filter to exclude pull requests where you've turned off notifications or where another team member is already looking into it. One way to achieve this is by using the `-review-requested` qualifier to exclude specific reviewers or teams. 

Here's how you can adjust your filter:

```markdown
is:open is:pr review-requested:huehnerlady -review-requested:@yourteam archived:false draft:false

**Reply by huehnerlady** [reply]
well the problem is it is 1 pull request that was requested and not the team in general. I do want to have the review requested via team in my pull request list, just not the one that I am not interested in, ergo where I turned off the notification

**Reply by akhilsamvarghese** [reply, marked as answer]
I understand the situation better now. Unfortunately, GitHub's filtering capabilities are somewhat limited in this context. While you can exclude PRs where you are directly requested, filtering out PRs where you've turned off notifications without excluding team requests as a whole is a bit tricky.

However, one workaround could be to **manually manage your notifications**. Here's what you can do:

1. **Turn off Notifications for Specific PRs**: If you’re not interested in a specific PR, you can manually unsubscribe from notifications for that particular PR. This way, it won’t clutter your notifications even if it still shows up in your filtered list.

2. **Use the `-review-requested` Qualifier for Yourself**: This excludes PRs where you are individually requested, but it will still include PRs where your team is requested:

   ```shell
   is:open is:pr -review-requested:huehnerlady review-requested:@yourteam archived:false draft:false

Unfortunately, this does mean you’ll need to manually sift through the list to ignore the ones you’ve turned off notifications for.

3. **Create a Custom Label:** You could also use a specific label, such as ignore-me, to mark the PRs you’re not interested in, and then filter them out of your list:
```shell
is:open is:pr review-requested:huehnerlady -label:ignore-me archived:false draft:false

**Reply by huehnerlady** [reply]
that is a shame. I was hoping there is a flag about that. But I guess then I have to live with that. Thanks :)
